crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -745,6 +745,72 @@ pub fn layer_norm_config(node: &Node) -> (LayerNormConfig, bool) {
     )
 }
 
+/// Create a PadConfig from the attributes of the node
+pub fn pad_config(node: &Node) -> PadConfig {
+    fn get_pads(node: &Node) -> Vec<usize> {
+        if node.inputs.len() != 3 {
+            panic!("Pad: must provide three inputs")
+        }
+
+        let input_dim = match &node.inputs.first().unwrap().ty {
+            ArgType::Tensor(tensor) => tensor.dim,
+            _ => panic!("Pad: Only tensor input is valid"),
+        };
+
+        let pads: Vec<usize> = match &node.inputs[1].value {
+            Some(Data::Int64s(shape)) => shape
+                .iter()
+                .map(|&x| {
+                    if x < 0 {
+                        // TODO: support negative pads
+                        panic!("Pad: Negative pad is not supported");
+                    }
+                    x as usize
+                })
+                .collect(),
+            _ => panic!("Pad: pads data type must be int64"),
+        };
+
+        if pads.len() != input_dim * 2 {
+            panic!("Pad: pads should be a 1D tensor of shape [2 * num_axes]");
+        }
+        // TODO: Burn's pad should support 1D tensor
+        if input_dim < 2 {
+            panic!("Pad: input tensor should be rank 2 or higher");
+        }
+
+        let left_index = input_dim - 1;
+        let top_index = input_dim - 2;
+        let right_index = pads.len() - 1;
+        let bottom_index = pads.len() - 2;
+        let index_list = [left_index, top_index, right_index, bottom_index];
+
+        for index in 0..pads.len() {
```

Comment:
Not sure why the CI didn't trigger at the PR level, but in the review mode I can see a suggestion:

```
[clippy] reported by reviewdog 🐶
error: the loop variable `index` is used to index `pads`
   --> crates/burn-import/src/onnx/op_configuration.rs:788:22
    |
788 |         for index in 0..pads.len() {
    |                      ^^^^^^^^^^^^^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_range_loop
    = note: `-D clippy::needless-range-loop` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::needless_range_loop)]`
help: consider using an iterator and enumerate()
    |
788 |         for (index, <item>) in pads.iter().enumerate() {
    |             ~~~~~~~~~~~~~~~    ~~~~~~~~~~~~~~~~~~~~~~~
```

Can you make the requested changes please? Otherwise CI will fail when we merge.

Reply:
Of course. Just pushed my changes.